docs/standard/attributes/writing-custom-attributes.md

```diff
@@ -19,7 +20,7 @@ ms.assetid: 97216f69-bde8-49fd-ac40-f18c500ef5dc
 ---
 # Writing Custom Attributes
 
-To design your own custom attributes, you do not need to learn many new concepts. If you are familiar with object-oriented programming and know how to design classes, you already have most of the knowledge needed. Custom attributes are essentially traditional classes that derive directly or indirectly from <xref:System.Attribute?displayProperty=nameWithType>. Just like traditional classes, custom attributes contain methods that store and retrieve data.  
+To design your custom attributes, you don't need to learn many new concepts. If you're familiar with object-oriented programming and know how to design classes, you already have most of the knowledge needed. Custom attributes are traditional classes that derive directly or indirectly from the <xref:System.Attribute?displayProperty=nameWithType>. Just like traditional classes, custom attributes contain methods that store and retrieve data.  
```

Comment:
Remove the extra full stop after "knowledge needed".

Reply:
@Lovina-Saldanha I'm unable to find the extra full stop. Could you please specify the code line?